Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/FileIO/DirectoryManager.cs

```diff
@@ -8,16 +10,64 @@ public class DirectoryManager : IDirectoryManager
 
         public bool Exists(string path) => Directory.Exists(path);
 
+        /// <summary>
+        /// This method mimics the behaviour of https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getrelativepath?view=net-6.0
+        /// </summary>
         public string GetRelativePath(string relativeTo, string path)
```

Comment:
Worth shimming the method as it is from System.IO?

https://github.com/dotnet/runtime/blob/09c1a1f7b0c477890b04912d8dd4f742f80faffc/src/libraries/System.Private.CoreLib/src/System/IO/Path.cs#L822

Along with providing clear attribution.

Reply:
I did think about this and discussed it with @normj. He suggested that we port the existing logic in the `dotnet-cli-extensions` to get the relative path.

Existing logic - https://github.com/aws/aws-extensions-for-dotnet-cli/blob/9639f8f4902349d289491b290979a3a1671cc0a5/src/Amazon.Common.DotNetCli.Tools/Utilities.cs#L91

@normj Do you want to reconsider porting the method verbatim (with citations) from System.IO?

Reply:
Even if we don't take System.IO change, add a reference to aws-extensions-for-dotnet-cli repo, so we are aware of the change has been taken straight from other place. Future updates will be easier.

Reply:
Fixed